iocore/net/I_NetVConnection.h

```diff
@@ -566,6 +573,21 @@ class NetVConnection : public VConnection
     is_transparent = state;
   }
 
+  /// Set the context and return context value
+  NetVConnectionContext_t
+  set_context(NetVConnectionContext_t context)
+  {
+    if (NET_VCONNECTION_UNSET == netvc_context)
+      netvc_context = context;
+    return netvc_context;
+  }
+  /// Get the context.
+  NetVConnectionContext_t
+  get_context() const
+  {
```

Comment:
Actually, I did reference the existing functions to name these two functions, such as "set_remote_addr" and "get_remote_addr".